cloudevents/sdk/marshaller.py

```diff
@@ -62,7 +62,7 @@ def FromRequest(
             raise exceptions.InvalidDataUnmarshaller()
 
         # Lower all header keys
-        headers = {key.lower(): value for key, value in headers.items()}
+        headers = {key.lower(): headers[key] for key in headers}
```

Comment:
What's the reason for this change?

Reply:
Probably a legacy change I made. I've rewritten it in a more recent commit.